src/Indigo/Contracts/HomebaseLite/Types/Self.hs

```diff
@@ -113,23 +104,3 @@ data MetadataConfig = MetadataConfig
 
 instance Default MetadataConfig where
   def = MetadataConfig "Homebase-Lite" "Offchain, decentralized voting system."
-
-myAnnOptions :: AnnOptions
-myAnnOptions = AnnOptions stripPrefix
-
--- this is cloned from "Morley.Michelson.Typed.Haskell.Doc"
--- [TODO: morley#759]: Remove this when 'dropPrefixThen' behaves the same, or this whole
--- manual tweaking becomes unnecessary.
-stripPrefix :: Text -> Text
-stripPrefix fieldName =
```

Comment:
Note: the new morley algorithms convert `sFA2Config` to `fa2Config` rather than `FA2Config` as was the case here. This is a breaking change. It's not hard to work around this to keep things consistent, but IIUC nobody is actually using this yet (or maybe ever)